x-pack/plugins/uptime/common/runtime_types/ping/ping.ts

```diff
@@ -38,7 +38,7 @@ export const MonitorType = t.intersection([
     name: t.string,
     timespan: t.partial({
       gte: t.string,
-      lte: t.string,
+      lt: t.string,
```

Comment:
This change looks unrelated. Was this intentional? If so it should go in its own PR.